### PR TITLE
WRN-19694: Fix isomorphic build is not working

### PR DESCRIPTION
**Pull request by MikyungKim**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
With the base branch, isomorphic build is failed.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Moved `document` related code to the guard block.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
To test, you need to use cli and dev-utils branch `feature/react-18`.

### Links
[//]: # (Related issues, references)
WRN-19694

### CODE REVIEW

File: samples/event-logger/src/index.js [hunk 1]
```diff
@@ -16,7 +12,11 @@ const appElement = (
 );
 
 if (typeof window !== 'undefined') {
+	const container = document.getElementById('root');
+	const root = createRoot(container);
+
 	root.render(appElement);
 }
 
+
```
Comment (jeonghee27) [inline samples/event-logger/src/index.js:21]:
```suggestion

```

File: samples/qa-fonts/src/index.js [hunk 2]
```diff
@@ -2,14 +2,13 @@ import {createRoot} from 'react-dom/client';
 
 import App from './App';
```
Comment (jeonghee27) [inline samples/qa-fonts/src/index.js:3]: I failed isomorphic pack on qa-fonts

```

Unable to generate prerender of app state HTML for en-US:
ReferenceError: document is not defined
    at Array.map (<anonymous>)
```

I guess there is problem on/qa-fonts/src/components/FontList/FontList.js
```
{document.fonts.check(font) ?
````

**Comment by jeonghee27** [comment]
qa-voice-control also has build fail

File: samples/qa-scroller/src/index.js [hunk 3]
```diff
@@ -2,14 +2,13 @@ import {createRoot} from 'react-dom/client';
```
Comment (jeonghee27) [inline samples/qa-scroller/src/index.js:2]: There is build error

Unable to generate prerender of app state HTML for en-US:
TypeError: Cannot read property 'undefined' of undefined

File: samples/event-logger/src/index.js (hunk 1 above)
Comment (MikyungKim) [reply samples/event-logger/src/index.js:21]: 👍

File: samples/qa-fonts/src/index.js (hunk 2 above)
Comment (MikyungKim) [reply samples/qa-fonts/src/index.js:3]: 🆗

File: samples/qa-scroller/src/index.js (hunk 3 above)
Comment (MikyungKim) [reply samples/qa-scroller/src/index.js:2]: `qa-voice-control` and `qa-scroller` needs more time to debug. But it's been errored with react17, so it seems not related to react18. Let's revisit this later.

**Pull request merged**